lms/djangoapps/grades/course_grade.py

```diff
@@ -289,3 +290,7 @@ def _compute_passed(grade_cutoffs, percent):
         nonzero_cutoffs = [cutoff for cutoff in grade_cutoffs.values() if cutoff > 0]
         success_cutoff = min(nonzero_cutoffs) if nonzero_cutoffs else None
         return success_cutoff and percent >= success_cutoff
+
+
+def _uniqueify_and_keep_order(iterable):
+    return OrderedDict([(item, None) for item in iterable]).keys()
```

Comment:
**Note to reviewers:** This helper was renamed to be clear that it's not just creating a `set` and moved down in the page to follow this module's step-down convention.